pupa/settings.py

```diff
@@ -9,6 +9,10 @@
 
 # scrape settings
 
+ENABLE_KAFKA = os.environ.get('ENABLE_KAFKA', "False").lower() == "true"
+KAFKA_REPORT_TOPIC = 'post-scrape-reports'
```

Comment:
It might make sense to split this into namespaced scrapper specif topics, i.e. "dc/post-scrape-reports", "federal-lobbying/post-scrape-reports". The same would apply for the forwarder.